sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/EventPosition.cs

```diff
@@ -27,15 +27,15 @@ public struct EventPosition : IEquatable<EventPosition>
         ///   which has not expired due to the retention policy.
         /// </summary>
         ///
-        public static EventPosition Earliest { get; } = new EventPosition { Offset = StartOfStream, IsInclusive = false };
+        public static EventPosition Earliest { get; } = new EventPosition { SequenceNumber = StartOfStream, IsInclusive = false };
```

Comment:
Only to move it back to `GlobalOffset` in the next PR.    😄

Reply:
I know hahaha 😢